frontends/web/src/routes/account/summary/accountssummary.tsx

```diff
@@ -61,6 +62,17 @@ interface BalanceRowProps {
     coinCode: accountApi.CoinCode;
 }
 
+const BuyCTAActual: FunctionalComponent<{balances?: Balances}> = ({balances}) => {
+    if (balances === undefined)
+        return null;
+    const balanceList = Object.entries(balances);
+    if (balanceList.some(entry => entry[1].available.amount !== '0'))
+        return null;
+    if ((balanceList.length === 1) && balanceList[0][1].available.unit === 'BTC')
+        return <BuyCTA code={balanceList[0][0]} unit={'BTC'} />;
+    return <BuyCTA />;
+};
```

Comment:
I was thinking how this could be simpler, but I think it is ok.. can you change the formatting to

```diff
diff --git a/frontends/web/src/routes/account/summary/accountssummary.tsx b/frontends/web/src/routes/account/summary/accountssummary.tsx
index b160edd5..d32635a0 100644
--- a/frontends/web/src/routes/account/summary/accountssummary.tsx
+++ b/frontends/web/src/routes/account/summary/accountssummary.tsx
@@ -62,14 +62,21 @@ interface BalanceRowProps {
     coinCode: accountApi.CoinCode;
 }
 
-const BuyCTAActual: FunctionalComponent<{balances?: Balances}> = ({balances}) => {
-    if (balances === undefined)
+type BuyCTAActualProps = {
+    balances?: Balances
+}
+
+const BuyCTAActual: FunctionalComponent<BuyCTAActualProps> = ({balances}) => {
+    if (balances === undefined) {
         return null;
+    }
     const balanceList = Object.entries(balances);
-    if (balanceList.some(entry => entry[1].available.amount !== '0'))
+    if (balanceList.some(entry => entry[1].available.amount !== '0')) {
         return null;
-    if ((balanceList.length === 1) && balanceList[0][1].available.unit === 'BTC')
+    }
+    if ((balanceList.length === 1) && balanceList[0][1].available.unit === 'BTC') {
         return <BuyCTA code={balanceList[0][0]} unit={'BTC'} />;
+    }
     return <BuyCTA />;
 };
 ```

Reply:
This component conditionally renders the CTA button and finds out if it is BTC only, I think we could hide it a bit and move to `info/buyCTA`.

Maybe name it `<AddBuyOnEmptyBalances balances={balances} />` ?

Reply:
Ok, i migrated that small component but had to export `Balances` interface. is that ok?

Reply:
yes thank you